vwifi.c

```diff
@@ -846,17 +846,102 @@ static int owl_delete_interface(struct owl_vif *vif)
     return 0;
 }
 
+/* Called by kernel when user decided to change the interface type. */
+static int owl_change_iface(struct wiphy *wiphy,
```

Comment:
Just curiosity, How does the user change the interface type? Which user space program could invoke this function?

Reply:
iw and hostapd can do.
With iw: `sudo iw dev owl0 set type ap`.
With hostapd, which will change the interface type via nl80211, and finally owl_change_iface() will be called.

Reply:
thanks for explaining